patches/hardhat+2.0.3.patch

```diff
@@ -0,0 +1,33 @@
+diff --git a/node_modules/hardhat/builtin-tasks/compile.js b/node_modules/hardhat/builtin-tasks/compile.js
```

Comment:
It'd be nice to include a rough explaination in the git commit message here.

Reply:
I added a bit more context in a squashed commit message for the PR.